tests/resources/backup/HydrogenODLPageCompilation_2022_01_01.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Using this HydrogenODL is probably now a good example as they are no longer generated on PROD I think.

Reply:
Did you mean that it would be better to have another example (like IEEEStandardDraftYANGPageCompilation)?

Reply:
Exactly

Reply:
Also maybe we should remove all ODL mentions in the sdo-analysis as it is not used currently.
- HydrogenODL
- HeliumODL
- LithiumODL